terraform_files/none/main.tf

```diff
@@ -113,7 +113,7 @@ module "workers" {
   source = "../baremetal_host"
   count  = var.worker_count
 
-  name           = "${var.cluster_name}-worker-${count.index}"
+  name           = count.index % 2 == 0 ? "${var.cluster_name}-worker-${count.index}" : "${var.cluster_name}-worker-secondary-${count.index}"
```

Comment:
So if you have 2 workers the first worker will be worker-secondary-1 and the second will be worker-2. Why?

Reply:
btw this is the names from the CI run's 
```
"name": "test-infra-cluster-8b3106c1-worker-0",
"name": "test-infra-cluster-8b3106c1-worker-secondary-1",
```

Reply:
Yes, the idea is to label the worker based on its network for easier debugging. Notice that this is what we do in maters already

Reply:
to this set the name if the host number is even or not.  
so I'm not sure if this is the result you wanted here

Reply:
Yes, to split the network hosts half-half. I just copied the code from masters